### DISCUSSION TITLE
175

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#140] [General]
# 潮流周刊第175期 - 有线耳机

封面图拍摄来源于上周精挑细选买的一个有线耳机，太太太好听了，51 回了一趟老家，奈何天天下雨，就只能躺平了，躺平听歌窗外下着雨也非常不错，此外这一期内容由于“躺平”的缘故，都是一些偏好玩的东西，少了一些技术。

https://weekly.tw93.fun/posts/175-%E6%9C%89%E7%BA%BF%E8%80%B3%E6%9C%BA

**Comment by FelixLyfe** [comment]
Apple music 是什么区的账号啊🤔

**Reply by tw93** [reply]
暂时还是国区的，一直订阅着稍微简单点儿

**Comment by TenTh0usand** [comment]
不过其实监听还是不适合日常听歌，询价/购买中高端耳机🎧可以联系我，本科生帮朋友代理各种乐器和音频设备，只询价也可以，我也搭了我的站点weekly.60004000.xyz

**Comment by lifei6671** [comment]
我用的就是Apple AirPods Max，降噪效果不过，真的是太沉了，戴时间长了耳朵疼，头也疼。

**Reply by tw93** [reply]
是的 这款颜值真yyds，不过就是太太太重了

**Comment by LavaCxx** [comment]
几年前买过一个索尼7506，也是上世纪的老监听耳机了，音质和我的其它设备比也能听出一耳朵区别。不过在宿舍我倾向于用音响外放，在不扰民的前提下还是更有氛围感的，而且也没有线、充电、耳朵累的问题。

**Reply by tw93** [reply]
是的是的

**Comment by CY-Christin** [comment]
我也尝试过有线的，不过是个入耳式耳机，森海塞尔的 ie300？具体型号记不清了。用起来是真的麻烦，想用就得先理线，有线的麻烦在我吃饭接水上厕所的时候体现的淋漓尽致，后来还是换回了 AirPods。

**Reply by tw93** [reply]
会有一点儿，有线我一般在室内使用，无线外出使用，当然假如不是太欣赏音乐那种，无线完全够用

**Comment by dox-xu** [comment]
感谢分享歌单
这耳机的空间音效是怎么回事，需要软件支持吗
